src/main/java/com/MyApp/budgetControl/domain/ServicesOrchestrator.java

```diff
@@ -0,0 +1,81 @@
+package com.MyApp.budgetControl.domain;
+
+import com.MyApp.budgetControl.domain.category.CategoryEntity;
+import com.MyApp.budgetControl.domain.category.CategoryRequestDTO;
+import com.MyApp.budgetControl.domain.category.CategoryResponseDTO;
+import com.MyApp.budgetControl.domain.category.CategoryService;
+import com.MyApp.budgetControl.domain.expense.ExpenseEntity;
+import com.MyApp.budgetControl.domain.expense.ExpenseRequestDTO;
+import com.MyApp.budgetControl.domain.expense.ExpenseResponseDTO;
+import com.MyApp.budgetControl.domain.expense.ExpensesService;
+import com.MyApp.budgetControl.domain.user.UserEntity;
+import com.MyApp.budgetControl.domain.user.UserRequestDTO;
+import com.MyApp.budgetControl.domain.user.UserResponseDTO;
+import com.MyApp.budgetControl.domain.user.UserService;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class ServicesOrchestrator {
+
+  private final CategoryService categoryService;
```

Comment:
A topic to check - architecture tests with Archunit library, It would be nice to write tests to verify that services are used just here and that there are no cases where one service uses another one inside, this way You can protect Your architecture decisions with tests

Reply:
First tests with Archunit added